Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -145,6 +149,14 @@ HRESULT InitializeGeometries() {
     }
 };
 
+HRESULT _CGPathGetGeometry(CGPathRef path, ID2D1Geometry** pGeometry) {
```

Comment:
>pGeometry [](start = 59, length = 9)

nit: if pGeometry is invalid, we end up returning S_OK.
adding a check for pGeoMetry and path should help.

RETURN_HR_IF_NULL(E_POINTER,pGeometry);
RETURN_HR_IF_NULL(E_POINTER,path);